frontend/src/index.tsx

```diff
@@ -44,6 +45,16 @@ createRoot(document.querySelector('#root') as HTMLElement).render(
                 path: AppRoute.SIGN_UP,
                 element: <Auth />,
               },
+              {
+                path: AppRoute.PROFILE,
+                element: (
+                  <ProtectedRoute>
+                    <SidebarWrapper>
+                      <UserProfile />
+                    </SidebarWrapper>
+                  </ProtectedRoute>
+                ),
```

Comment:
Let's rename it to dashboard, also it should be our root component, so put it at the top instead of root
```suggestion
                path: AppRoute.ROOT,
                element: (
                  <ProtectedRoute>
                    <SidebarWrapper>
                      <Dashboard />
                    </SidebarWrapper>
                  </ProtectedRoute>
                ),
```

Reply:
done